src/test/java/org/influxdb/InfluxDBTest.java

```diff
@@ -23,177 +24,233 @@
 
 /**
  * Test the InfluxDB API.
- * 
+ *
  * @author stefan.majer [at] gmail.com
- * 
  */
 @Test
 public class InfluxDBTest {
 
-	private InfluxDB influxDB;
```

Comment:
Please do not reformat code which is untouched. Can you please fix this, otherwise im fine with this PR and will merge it.